File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ArchaicAmulet.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class ArchaicAmulet extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_archaic_amulet_name;
        this.idDescription = R.string.accessory_archaic_amulet_description;
        this.idEffect = R.string.accessory_archaic_amulet_effect;
        this.idImage = R.drawable.archaic_amulet;
        this.price = 806L;
        this.livingCompanionBonusDamage = 40;
        this.maxHp = 140;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ArchaicAmulet.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class ArchaicAmulet : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_archaic_amulet_name
        idDescription = R.string.accessory_archaic_amulet_description
        idEffect = R.string.accessory_archaic_amulet_effect
        idImage = R.drawable.archaic_amulet
        price = 806L
        livingCompanionBonusDamage = 40
        maxHp = 140
    }
}
